src/js/components/stock-movement/outbound/StockMovementOutboundTable.jsx

```diff
@@ -129,19 +131,20 @@ const StockMovementOutboundTable = ({
       Header: <Translate id="react.stockMovement.column.status.label" defaultMessage="Status" />,
       accessor: 'status',
       fixed: 'left',
-      width: 150,
+      width: 170,
       sortable: false,
-      Cell: (row) => {
-        const status = _.find(requisitionStatuses, _.matchesProperty('id', row.value));
-        return (
-          <TableCell
-            {...row}
-            tooltip
-            tooltipLabel={getStatusTooltip(row.value)}
-          >
-            <StatusIndicator variant={status?.variant} status={status?.label} />
-          </TableCell>);
-      },
+      Cell: row => (
+        <TableCell
+          {...row}
+          tooltip
+          tooltipLabel={getStatusTooltip(row.value)}
+        >
+          <StatusIndicator
+            variant={row?.original?.statusVariant}
+            status={translate(`react.requisitionStatus.enum.${row?.original?.status}`, row?.original?.status)}
```

Comment:
please make sure that the label will change when you change the language - I remember having difficult times with `translate` behavior when changing the language on list pages - most of the times I was just deciding to translate what I needed on the backend.

Reply:
Eh, you're right, it doesn't work